docs/pages/machine-id/manifesto.mdx

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Would it make sense to move this content to the Introduction page, where we include general information about the scope and purpose of Machine ID? I'm wondering if that would make this content more discoverable than adding it to a manifesto page.

If that makes the Introduction too long, we could consider moving some of the more detailed architectural information to a separate Architecture page.

Reply:
I'm wary of putting this onto the introduction page, whilst this is nice reading, and interesting to those who want to know more about what motivates our decisions and direction of Machine ID, for those seeking to deploy or troubleshoot Machine ID, it would be distracting.

Reply:
If we're worried about this taking up side bar space, I'm happy to drop it from there and just have it linked to within the introduction. The main idea here is just to have something we can share when people ask about our vision - organic discovery is a plus but I don't think is critical.

Reply:
Taking up sidebar space should be fine for now, though this quarter, we're moving to a model in which we reserve the sidebar for limited links to high-value pages and use "All Topics" pages (similar to this [very rough initial example](https://docs-dl0gup7uh-goteleport.vercel.app/docs/reference/all-topics/)) to link to pages in a particular section. 

I'm okay with trying out a separate Manifesto page to start. We can review it in a month or so to see if it's getting the kind of traffic we want it to get!